environment.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
   # - importlib-metadata   # [py<38]
   - ruamel.yaml
   - snakemake-minimal >= 7.22.0
+  - openjdk >= 17.0.0
```

Comment:
It’s strange this is required because the Bioconda recipe already has a [requirement on openjdk>=17](https://github.com/bioconda/bioconda-recipes/blob/955bd9a368386e616f5ac81df512474807f4c7ea/recipes/picard/meta.yaml#L18). Can you add a comment here to clarify that this line should not be needed? 

We should normally only list the direct dependencies and let Conda figure out what needs to be installed. If the above line is actually needed, it should be considered a temporary workaround until the recipes or Conda themselves are fixed.

Reply:
Yes, it is strange, I also checked the Picard Bioconda recipe, so maybe it is some problem solving the environment, I could reproduce the problem on the GitHub test creating the environment fresh locally on my computer. Somehow the openjdk version is not correctly resolved, because it still installs `openjdk 11.0.15` and then picard complains. At least, mamba does. I am checking if creating the environment with conda resolves this, but it is still running, so that is also not very desirable for the CI testing, even if it works.

Reply:
Just an update comment that conda also installs an inferior version of `openjdk 11.0.1` when resolving the environment, I have no idea why. So I will put a comment on the environment as you suggest for the time being.

Reply:
Oh, I think I found it: There exists an [older version of the Bioconda recipe](https://github.com/bioconda/bioconda-recipes/commit/bc603ffcfc2c003c17d71616622d5e66ab6dc556) for which the dependency is `openjdk>=8`. So probably the solver picks that one (with build number 0). I think we should be able to fix the problem by depending on `picard>=3.0.0=1` (not sure if that spelling is how to specify build numbers in version specifiers).

Reply:
I tried this now, but even though it parses it correctly: 

```
Looking for: ... [ "picard[version='>=3.0.0',build=1]" ...
``` 

then it prints an error: 

```
└─ picard >=3.0.0 1 does not exist (perhaps a typo or a missing channel).
``` 

So it seems something is not totally up-to-date where the recipes are pulled from? It is very odd because that change in conda recipes was made 6 months ago.

Reply:
Ok, the above specifier doesn’t work because the full build string needs to be given: `picard>=3.0.0=hdfd78af_1`.

Maybe after all it’s easier to write `openjdk>=17.0.0` with a comment as you did originally. But the comment could state that the explicit `openjdk` dependency can be removed as soon as the picard dependency is bumped to 3.0.1 or later.